community/releases/next-release-preview/index.md

```diff
@@ -12,3 +12,7 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 1.26
+
+### (Breaking Change) Suffix no longer added to jobs created by Kubernetes Run Job stage
+
+As a followup to [this change in Spinnaker version 1.22](https://spinnaker.io/community/releases/versions/1-22-0-changelog#breaking-change-suffix-no-longer-added-to-jobs-created-by-kuber), Spinnaker 1.26 removes the `kubernetes.jobs.append-suffix` flag.  Note that the default value of this flag was false in Spinnaker version >= 1.22.  To continue having a random suffix added to the job name, set the `metadata.generateName` field instead of `metadata.name`, which causes the [Kubernetes API](https://kubernetes.io/docs/reference/using-api/api-concepts/#generated-values) to append a random suffix to the name.
```

Comment:
Sorry, can you tell me the full path you're suggesting?

Reply:
`guides/user/kubernetes-v2/run-job-manifest/index.md`

Reply:
Sure...though I just realized these notes didn't make it to https://spinnaker.io/community/releases/versions/1-26-0-changelog#curated-changlog...

Reply:
Sorry, this is all finally sinking in.  Doesn't the rest of this PR update `guides/user/kubernetes-v2/run-job-manifest/index.md` appropriately?  So the thing I need to do to get this merged is remove the changes to `community/releases/next-release-preview/index.md` ?

Reply:
I think I've done this now...

Reply:
@link108 any chance you can help get these notes added to https://spinnaker.io/community/releases/versions/1-26-0-changelog#curated-changlog ? Sorry I didn't notice it didn't get merged in time.

Reply:
Yeah, sorry about that. This PR slipped through the cracks. We should have noticed earlier and let you know that it should have been 2 PRs - one for the Next Release info and another for the docs update.

Reply:
@link108 

```
(Breaking Change) Suffix no longer added to jobs created by Kubernetes Run Job stage

As a followup to [this change in Spinnaker version 1.22](https://spinnaker.io/community/releases/versions/1-22-0-changelog#breaking-change-suffix-no-longer-added-to-jobs-created-by-kuber), Spinnaker 1.26 removes the `kubernetes.jobs.append-suffix` flag.  Note that the default value of this flag was false in Spinnaker version >= 1.22.  To continue having a random suffix added to the job name, set the `metadata.generateName` field instead of `metadata.name`, which causes the [Kubernetes API](https://kubernetes.io/docs/reference/using-api/api-concepts/#generated-values) to append a random suffix to the name.
```